frontend/src/components/PageHeader/PageHeader.module.css

```diff
@@ -1,10 +1,18 @@
-.header{
+.headerContainer{
     border: 1px solid #666666;
+    border-radius: 5px;
     padding: 1.5%;
     margin-bottom: 2%;
     background-color: #1C1C1C;
     padding-left: 3%;
     padding-right: 3%;
     color:#fff;
     text-align: left;
+    font-size: 1.5em;
```

Comment:
It would be better to set font-size in pixels, for example 30px.

Reply:
switched to pixels